src/Domain/ExcisionSphere.cpp

```diff
@@ -29,6 +32,56 @@ ExcisionSphere<VolumeDim>::ExcisionSphere(
   }
 }
 
+template <size_t VolumeDim>
+ExcisionSphere<VolumeDim>::ExcisionSphere(
+    const ExcisionSphere<VolumeDim>& rhs) {
+  radius_ = rhs.radius_;
+  center_ = rhs.center_;
+  abutting_directions_ = rhs.abutting_directions_;
+  if (rhs.grid_to_inertial_map_ == nullptr) {
+    grid_to_inertial_map_ = nullptr;
+  } else {
+    grid_to_inertial_map_ = rhs.grid_to_inertial_map_->get_clone();
+  }
+}
+
+template <size_t VolumeDim>
+ExcisionSphere<VolumeDim>& ExcisionSphere<VolumeDim>::operator=(
+    const ExcisionSphere<VolumeDim>& rhs) {
+  radius_ = rhs.radius_;
+  center_ = rhs.center_;
+  abutting_directions_ = rhs.abutting_directions_;
+  if (rhs.grid_to_inertial_map_ == nullptr) {
+    grid_to_inertial_map_ = nullptr;
+  } else {
+    grid_to_inertial_map_ = rhs.grid_to_inertial_map_->get_clone();
+  }
+  return *this;
+}
+
+template <size_t VolumeDim>
+void ExcisionSphere<VolumeDim>::inject_time_dependent_maps(
+    std::unique_ptr<
+        domain::CoordinateMapBase<Frame::Grid, Frame::Inertial, VolumeDim>>
+        moving_mesh_grid_to_inertial_map) {
+  grid_to_inertial_map_ = std::move(moving_mesh_grid_to_inertial_map);
+}
+
+template <size_t VolumeDim>
+const domain::CoordinateMapBase<Frame::Grid, Frame::Inertial, VolumeDim>&
+ExcisionSphere<VolumeDim>::moving_mesh_grid_to_inertial_map() const {
+  // This is an ERROR rather than an ASSERT because we don't expect to be
+  // calling this as often as the coordinate maps in the Blocks, so always
+  // checking shouldn't be costly (and we wrap in an unlikely for good measure).
+  if (UNLIKELY(grid_to_inertial_map_ == nullptr)) {
+    ERROR(
```

Comment:
same comment about user vs programmer error

Reply:
I've messed this up when coding things into the domain creator and then an ASSERT wasn't hit in release mode and I got a segfault/UB. If this was going to be called super often like in the RHS, I'd make it an ASSERT, but this will only be called on events so it shouldn't be slow. And for that reason, I'd always want the extra protection.

Reply:
Just test things in Debug mode, that's what it's for :) And that's why the distinction between ERROR and ASSERT exist.